src/main/resources/de/nerdclubtfg/signalbot/components/defaultConfig.json

```diff
@@ -2,7 +2,8 @@
   "plugins": {
     "echo": true,
     "fefe" : true,
-    "plugins" : true
+    "plugins" : true,
```

Comment:
I had problems with GSON (google framework for json) generating null entries in the HashMap. Is this also true for FasterXML?